iterative/dvc-task
# dvc-10427 Fix Celery Monitor Shutdowns

Related to:

- https://github.com/iterative/dvc/issues/10427

Specifically, check [this comment](https://github.com/iterative/dvc/issues/10427#issuecomment-2323211461) as it contains the insight about what's going on.

## How to Test this
It's somewhat tricky to set up everything to verify the solution, so I have created a PR here to ease the process:

- https://github.com/nablabits/example-get-started-experiments/pull/1
